android/build.gradle

```diff
@@ -44,23 +44,26 @@ rootProject.allprojects {
 apply plugin: 'com.android.library'
 
 def DEFAULT_PLAY_SERVICES_BASE_VERSION = '16.0.1'
-def READER_SDK_VERSION = '[1.5.1, 2.0)'
+def READER_SDK_VERSION = '[1.6.1, 2.0)'
 
 android {
-    compileSdkVersion 29
+    compileSdkVersion 32
```

Comment:
The ReaderSDK only support up to 31

- [ ] downgrade android version to 31

Reply:
I put max version of SDK that reader sdk can work with without issues. Official docs saying that it doesn't work with SDK higher than 29 though) I tried sdk 33 but reader sdk has issues with storage permissions

Reply:
Yes, we don't officially support 32 and above. that's why 31 is the best so far.

Reply:
Hm, wait, it's a compile SDK, not a target SDK) I think it can be even set to 33

Reply:
Unless you think it can lead to any issues?

Reply:
Or you would like to keep it the same with targetSdk?

Reply:
You are right. Let's keep target Sdk Version at 31, then we should be good.

Reply:
updated target SDK value, let me know if you want to change compile SDK as well